tests/Fixtures/ObjectWithEnums.php

```diff
@@ -12,32 +12,34 @@
 class ObjectWithEnums
```

Comment:
Just thinking out loud - how it is going to work with Attributes? I guess we might miss some test cases for it but it might be used by some users as well.

Reply:
I believe it is just missing tests for Attributes on this particular level. I've said goodbye to annotations quite a while back, so I hit this while using Attributes and verified that for me this solution works.